### PR TITLE
Removed in 0.15 (Removed)

### LINKED ISSUE #4402: Remove the RemovedInKALite_v015_Warning warning
image

### LINKED ISSUE #4463: Create RemovedIn0_16 warning for Python 2.6
Since we're removing Python 2.6, we should create a proper removal warning for 0.15

### PR DESCRIPTION
**Pull request by rtibbles**
Fixes #4402 in the most conservative way possible.
Also resolves #4463.

Summary of changes:
* Delete the runcherrypy server management command.
* Punt settings refactors to a removed in 0.16 warning (@benjaoming - I don't think there is time to complete the settings refactor before 0.15 release, if you have a PR for it that you have been keeping up your sleeve, I would love to see it!)
* Adds a removed in 0.16 warning in setup for Python 2.6.

### CODE REVIEW

File: kalite/distributed/management/commands/setup.py [hunk 1]
```diff
@@ -30,6 +30,8 @@
 from kalite.contentload.settings import KHAN_ASSESSMENT_ITEM_ROOT, OLD_ASSESSMENT_ITEMS_LOCATION
 from kalite.topic_tools.settings import CHANNEL
 
+from kalite.shared.warnings import RemovedInKALite_v016_Warning
```
Comment (benjaoming) [inline kalite/distributed/management/commands/setup.py]: :+1: thanks!

File: kalite/distributed/management/commands/setup.py [hunk 2]
```diff
@@ -240,6 +242,12 @@ def handle(self, *args, **options):
         if sys.version_info < (2, 7, 9):
             logging.warning(
                 "It's recommended that you install Python version 2.7.9. Your version is: %d.%d.%d\n" % sys.version_info[:3])
+            if sys.version_info < (2, 6):
```
Comment (benjaoming) [inline kalite/distributed/management/commands/setup.py]: <= ;)

Comment (benjaoming) [inline kalite/distributed/management/commands/setup.py]: Actually.... `< (2, 7)` come to think of it :D

**Comment by benjaoming** [comment]
Well I would prefer keeping the 0.15 warnings so it's easy to see what we originally intended to remove. These are things we promised to remove, we should stick to those promises... not like removing guantanamo.

File: kalite/distributed/management/commands/setup.py (hunk 2 above)
Comment (rtibbles) [inline kalite/distributed/management/commands/setup.py]: Oh, yeah, good catch!

**Comment by rtibbles** [comment]
Yeah, I would prefer to remove them, but unlike Guantanamo, we're not holding anyone without charge.

**Comment by aronasorman** [comment]
Looks good to me. If there are no complaints I'll merge this in later this afternoon PST.

**Pull request merged**